config/initializers/assets.rb

```diff
@@ -1,10 +1,7 @@
 Rails.application.config.assets.precompile += %w(
-  miq_browser_detect.js
   jquery-1.8/jquery.js jquery_overrides.js
   miq_policy/import.js
   novnc-rails.js miq_novnc.js
   spice-html5.js spiceHTML5/spicearraybuffer.js
-
-  xml_display.css
-  container_topology.css service_dialogs.css import_policy.css vmrc.css
```

Comment:
#6081 , only container_topology and import_policy needed tweaking, vmrc doesn't affect anything but and service_dialogs is ok.